Calculator.sln

```diff
@@ -0,0 +1,30 @@
+﻿

```

Comment:
Hi there! You've created the project in the main repository folder. I think I need to update the code review article to make it more clear, but you need to create your project as a subfolder of the repository, as per these instructions 
![image](https://user-images.githubusercontent.com/108711476/210284343-ca779ac1-238d-4c82-b7f0-7cdc1acaceee.png) in this article: https://www.thecsharpacademy.com/article/52

Let me know if you need help, dealing with source control can be tricky.